sidebars.js

```diff
@@ -19,8 +19,18 @@ module.exports = {
         "overview",
         {
           type: "link",
-          label: "PDF Download",
-          href: "/pdf",
+          label: "Core Concepts",
+          href: "/docs/basic/core-concepts",
+        },
+        {
+          type: "link",
+          label: "Server Installation",
+          href: "/docs/basic/server-installation",
+        },
+        {
+          type: "link",
+          label: "Casdoor SSO",
+          href: "/docs/basic/casdoor-sso",
```

Comment:
You can define sidebar like this:

```js
module.exports = {
  tutorialSidebar: [
    {
      type: "category",
      label: "The Basics",
      link: { type: "generated-index" },
      items: [
        "overview",
        "basic/core-concepts",
        "basic/server-installation",
        "basic/casdoor-sso",
      ],
    },
  ],
};
```

For more information, please visit https://docusaurus.io/docs/sidebar/items

Reply:
It works fine on my machine, and the [preview](https://deploy-preview-7--casibase.netlify.app/docs/basic/core-concepts/) works fine too.

You can run `yarn clear` and try again

Reply:
That worked perfectly! Thank you so much for your assistance!